app/helpers/application_helper.rb

```diff
@@ -36,4 +36,20 @@ def alert_class(alert_type)
     }.with_indifferent_access.fetch(alert_type, alert_type.to_s)
     "alert-#{alert_type}"
   end
+  def react_component(component_name, props={}, html_options = {}, render_order: 0)
```

Comment:
I found myself gravitating towards https://github.com/renchap/webpacker-react to have this view helper.